.github/workflows/release.yaml

```diff
@@ -1,25 +1,64 @@
 name: Release Charts
 
 on:
+  pull_request:
+    branches:
+      - master
   push:
     branches:
       - master
+      - release*
 
 jobs:
-  release:
+  lint-test:
```

Comment:
How about separating into three jobs: lint, test and release?  And then we can define the dependencies among the jobs with `jobs.<job_id>.needs`. Refer to https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobs.

Reply:
updated